### ISSUE TITLE
Safari save as image - no pop-up window [browser support]

### ISSUE THREAD

**Issue by miktoki** [ESA-VirES/WebClient-Framework#50]
When attempting to save an image using Safari, instead of a pop-up window opening or new tab appearing, the image opens in the window, effectively exiting the application.

**Comment by santilland** [comment]
Ok, so i have been reading up into this issue and it seems Safari handles things a bit differently. It seems the HTML5 saveAs method is not fully supported (and the additional possible download attribute is also ignored). The only possible consistent solution i have found is the use of FileSaver library:
https://github.com/eligrey/FileSaver.js/
Which brings support for multiple browsers and kind of handles the Safari download. The best that can be done is that a new tab is opened with the image and the user needs to save the image "manually". It is possible to trigger a download (or saveAs) event using a different mime/type (kind of confusing the browser) but it seems this could maybe make the download not work at all for some Safari users:
https://github.com/eligrey/FileSaver.js/issues/12#issuecomment-47247096

So in conclusion i think opening the image in a new tab is the best and only currently possible solution for Safari browser. This update was introduced in commit 0e01b2db12d680dce0feb7cfd2bd9affcd266e3b

I propose to close the issue and if at some point there are some developments or new workaround or techniques found for Safari we can revisit the issue.
